mysql-test/suite/innodb/t/percona_changed_page_bmp_1364315-master.opt

```diff
@@ -0,0 +1 @@
+--innodb_track_changed_pages=1 --innodb_changed_pages=FORCE
```

Comment:
These options can be pushed to the testcase restart where it sets --innodb-data-home-dir, removing extra MTR server start